units/ArmBots/T2/armmark.lua

```diff
@@ -28,6 +28,7 @@ return {
 		onoffable = false,
 		radardistance = 2200,
 		script = "Units/ARMMARK.cob",
+		seismicsignature = 0,
```

Comment:
Why is this part of this PR?

Reply:
Other glitches that needed patching too

Reply:
Why is (patching other glitches that need patching too) part of this PR?

Reply:
They're chronologically related patches. Making four different PRs for some one-line changes would be insane, especially considering the semi chaos that usually passes for git protocol here.  The PR is basically my bugfix digest, Liches are just the headline, as the description outlines.

Reply:
> Making four different PRs for some one-line changes would be insane

no, it's absolutely normal

Reply:
Were we a paragon of git procedures I'd agree, or were the changes more substantial/possibly error prone, but it doesn't seem more wrong than anything else that goes on. Not having to make 4 more branches in my absolute spider web of a repo, and then chase around 4 different PRs, seems vastly preferable to possible PR purity concerns.

Reply:
4 is hyperbole, 1 extra for unrelated stuff is sensible.